lucene/core/src/java/org/apache/lucene/index/MergePolicy.java

```diff
@@ -542,13 +542,13 @@ public boolean useCompoundFile(SegmentInfos infos, SegmentCommitInfo mergedInfo,
       return false;
     }
     if (getNoCFSRatio() >= 1.0) {
-      return true;
+      return false;
     }
     long totalSize = 0;
     for (SegmentCommitInfo info : infos) {
       totalSize += size(info, mergeContext);
     }
-    return mergedInfoSize <= getNoCFSRatio() * totalSize;
+    return false;
```

Comment:
why are we changing existing code behavior?

Reply:
this thing also for remove compound file formation so we can see our file sizes more accurately